### ISSUE TITLE
Add ‘Okina to Character Set to Ensure Hawaiian Language Support

### ISSUE THREAD

**Issue by ThomasJockin** [google/roboto#348]
‘Okina is a required glyph for the Hawaiian language. The Google Fonts Glyphs Set includes the ‘Okina glyph (U+02BB). However, this font currently does not cover the ‘Okina glyph when it does cover the other required glyphs, /Amacron, /Emacron, etc
<img width="651" alt="Screen Shot 2022-07-06 at 9 30 48 AM" src="https://user-images.githubusercontent.com/16943529/177561893-38f702b4-b8c6-4b2d-9ddf-4f62d2d3c5dd.png">

**Comment by dylnclrk** [comment]
Did you check the "hinted" version of the fonts in this repo? The version from google fonts website didn't have the ‘Okina character, but the hinted version seems to include it.

**Comment by fitojb** [comment]
Indeed, what happens is just that the Google Fonts platform’s subsets don’t include the ‘okina.
